example/lib/main.dart

```diff
@@ -1,10 +1,20 @@
-import 'package:flutter/material.dart';
 import 'dart:async';
 
-import 'package:flutter/services.dart';
 import 'package:customer_io/customer_io.dart';
+import 'package:customer_io/customer_io_config.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  await CustomerIo.initialize(
+    config: CustomerIOConfig(
+      siteId: "",
+      apiKey: "",
+    ),
```

Comment:
Can we make the `CustomerIO.initalize()` function for the flutter SDK match the API of our native SDKs? 

By providing the siteId and apiKey as parameters of `initialize()` instead of inside of a config object? 

```suggestion
  CustomerIO.initialize(
      siteId: "",
      apiKey: "",
   ),
```